.gitignore

```diff
@@ -204,6 +204,9 @@ node_modules/
 /ajax/libs/slim.js/1.*/
 /ajax/libs/slim.js/2.[0-5].*/
 /ajax/libs/europa/4.0.0-alpha*/
+/ajax/libs/inferno-compat/*-alpha*/
+/ajax/libs/inferno-compat/*-beta*/
+/ajax/libs/inferno-compat/*-rc*/
```

Comment:
Don't we just want to exclude the dummy versions especially those version string contains checksum?